docker/worker/Dockerfile

```diff
@@ -45,10 +83,13 @@ RUN pip3 install pip --upgrade
 RUN pip3 install requests --upgrade
 RUN pip3 install urllib3 cryptography --upgrade
 
-# Install third-party worker dependencies
+# Install third-party dependencies
```

Comment:
Can address this now or save for a future clean up if it is the case, but do you think anytime we run pip install here we should just add to the requirements file instead?  Same goes for the `requests`, `urllib3`, `cryptography` deps above.